tools/clang/test/CodeGenSPIRV/switch-stmt.ifstmt.hlsl

```diff
@@ -0,0 +1,322 @@
+    case 1:
+      b=1;
+      break;
+    case 2:
+      b=2;
+  }
+
+
+  //////////////////////////////////////////////
+  // DefaultStmt in the middle of other cases //
+  //////////////////////////////////////////////
+
+// CHECK-NEXT: [[a2:%\d+]] = OpLoad %int %a
+// CHECK-NEXT: [[is_a_10:%\d+]] = OpIEqual %bool [[a2]] %int_10
+// CHECK-NEXT: OpSelectionMerge %if_merge_1 None
+// CHECK-NEXT: OpBranchConditional [[is_a_10]] %if_true_1 %if_false_1
+// CHECK-NEXT: %if_true_1 = OpLabel
+// CHECK-NEXT: OpStore %b %int_1
+// CHECK-NEXT: OpStore %b %int_0
+// CHECK-NEXT: OpStore %b %int_2
+// CHECK-NEXT: OpBranch %if_merge_1
+// CHECK-NEXT: %if_false_1 = OpLabel
+// CHECK-NEXT: [[a3:%\d+]] = OpLoad %int %a
+// CHECK-NEXT: [[is_a_20:%\d+]] = OpIEqual %bool [[a3]] %int_20
+// CHECK-NEXT: OpSelectionMerge %if_merge_2 None
+// CHECK-NEXT: OpBranchConditional [[is_a_20]] %if_true_2 %if_false_2
+// CHECK-NEXT: %if_true_2 = OpLabel
+// CHECK-NEXT: OpStore %b %int_2
+// CHECK-NEXT: OpBranch %if_merge_2
+// CHECK-NEXT: %if_false_2 = OpLabel
+// CHECK-NEXT: OpStore %b %int_0
+// CHECK-NEXT: OpStore %b %int_2
+// CHECK-NEXT: OpBranch %if_merge_2
+// CHECK-NEXT: %if_merge_2 = OpLabel
+// CHECK-NEXT: OpBranch %if_merge_1
+// CHECK-NEXT: %if_merge_1 = OpLabel
+  [branch] switch(a) {
+    case 10:
+      b=1;
+    default:
+      b=0;
+    case 20:
+      b=2;
+      break;
+  }
+
+  ///////////////////////////////////////////////
+  // Various CaseStmt and BreakStmt topologies //
+  // DefaultStmt is the last statement         //
+  ///////////////////////////////////////////////
+
+// CHECK-NEXT: [[d0:%\d+]] = OpLoad %int %d
+// CHECK-NEXT: [[is_d_1:%\d+]] = OpIEqual %bool [[d0]] %int_1
+// CHECK-NEXT: OpSelectionMerge %if_merge_3 None
+// CHECK-NEXT: OpBranchConditional [[is_d_1]] %if_true_3 %if_false_3
+// CHECK-NEXT: %if_true_3 = OpLabel
+// CHECK-NEXT: OpStore %b %int_1
+// CHECK-NEXT: [[foo:%\d+]] = OpFunctionCall %int %foo
+// CHECK-NEXT: OpStore %c [[foo]]
+// CHECK-NEXT: OpStore %b %int_2
+// CHECK-NEXT: OpBranch %if_merge_3
+// CHECK-NEXT: %if_false_3 = OpLabel
+// CHECK-NEXT: [[d1:%\d+]] = OpLoad %int %d
+// CHECK-NEXT: [[is_d_2:%\d+]] = OpIEqual %bool [[d1]] %int_2
+// CHECK-NEXT: OpSelectionMerge %if_merge_4 None
+// CHECK-NEXT: OpBranchConditional [[is_d_2]] %if_true_4 %if_false_4
+// CHECK-NEXT: %if_true_4 = OpLabel
+// CHECK-NEXT: OpStore %b %int_2
+// CHECK-NEXT: OpBranch %if_merge_4
+// CHECK-NEXT: %if_false_4 = OpLabel
+// CHECK-NEXT: [[d2:%\d+]] = OpLoad %int %d
+// CHECK-NEXT: [[is_d_3:%\d+]] = OpIEqual %bool [[d2]] %int_3
+// CHECK-NEXT: OpSelectionMerge %if_merge_5 None
+// CHECK-NEXT: OpBranchConditional [[is_d_3]] %if_true_5 %if_false_5
+// CHECK-NEXT: %if_true_5 = OpLabel
+// CHECK-NEXT: OpStore %b %int_3
+// CHECK-NEXT: OpBranch %if_merge_5
+// CHECK-NEXT: %if_false_5 = OpLabel
+// CHECK-NEXT: [[d3:%\d+]] = OpLoad %int %d
+// CHECK-NEXT: [[t:%\d+]] = OpLoad %int %t
+// CHECK-NEXT: [[is_d_eq_t:%\d+]] = OpIEqual %bool [[d3]] [[t]]
```

Comment:
This will also be taken care of when issue #110 is resolved i believe. Will address in a follow-up CL.